templates/cluster_resources.j2

```diff
@@ -58,17 +58,17 @@ primitive rsc_ip_{{ sid }}_ERS{{ ers_instance }} ocf:suse:aws-vpc-move-ip \
 {%- elif cloud_provider == "google-cloud-platform" %}
 
 # This stonith resource will be duplicated for each node in the cluster
-primitive rsc_gcp_stonith_{{ sid }}_HDB{{ ascs_instance }}_{{ grains['host'] }} stonith:fence_gce \
+primitive rsc_gcp_stonith_{{ sid }}_{{ grains['host'] }} stonith:fence_gce \
     params action=reboot plug={{ grains['host'] }} pcmk_host_map="{{ grains['host'] }}:{{ grains['host'] }}" \
     meta target-role=Started
 
-primitive rsc_ip_{{ sid }}_ASCS{{ instance }} ocf:heartbeat:gcp-vpc-move-route \
+primitive rsc_ip_{{ sid }}_ASCS{{ ascs_instance }} ocf:heartbeat:gcp-vpc-move-route \
```

Comment:
Should there be underscore _ before instance numbers . Same with ERS.

Reply:
@Simranpal We don't put it usually to be honest